src/models/teacher.ts

```diff
@@ -0,0 +1,34 @@
+import jagql from '@jagql/framework'
+import {getHandler} from '../handlers/sqlHandler';
+import {Batch} from './batch'
+
+const Joi = jagql.Joi
+const handler = getHandler()
+
+export interface Teacher {
+    name: string
+    id: string
+    contactNo?: string
+    email?: string
+    batches?: Batch[]
```

Comment:
similarly batches entiy should have `teachers: Teachers[]` field too now right ?